docs/raster_datasets.md

```diff
@@ -0,0 +1,65 @@
+# Raster datasets
+
+OQT operates on raster datasets stored on disk.
+
+
+## Location (Data directory)
+
+The raster files are located at `$XDG_DATA_HOME/oqt/`. If the environment variable `$XDG_DATA_HOME` is unset following path is used instead: `~/.local/share/oqt`
+
+The location can be changed by setting the environment variable `$OQT_DATA_DIR`.
+
+
+## Setup
+
+The following section will describe how to set up the individual raster datasets.
+
+
+### GHS-BUILT R2018A
+
+Information:
+- https://ghsl.jrc.ec.europa.eu/ghs_bu2019.php
+- product: GHS-BUILT, epoch: 2015, resolution: 1 km, coordinate system: Mollweide.
+
+Setup steps:
+1. Download the global raster as single file at [https://cidportal.jrc.ec.europa.eu/ftp/jrc-opendata/GHSL/GHS_BUILT_LDSMT_GLOBE_R2018A/GHS_BUILT_LDS2014_GLOBE_R2018A_54009_1K/V2-0/GHS_BUILT_LDS2014_GLOBE_R2018A_54009_1K_V2_0.zip](https://cidportal.jrc.ec.europa.eu/ftp/jrc-opendata/GHSL/GHS_BUILT_LDSMT_GLOBE_R2018A/GHS_BUILT_LDS2014_GLOBE_R2018A_54009_1K/V2-0/GHS_BUILT_LDS2014_GLOBE_R2018A_54009_1K_V2_0.zip)
+2. Extract archive
+3. Move `GHS_BUILT_LDS2014_GLOBE_R2018A_54009_1K_V2_0.tif` to the OQT data directory
+
+
+### GHS-POP R2019A
+
+Information:
+- https://ghsl.jrc.ec.europa.eu/ghs_pop2019.php
+- product: GHS-POP, epoch: 2015, resolution: 1 km, coordinate system: Mollweide.
+
+Setup steps:
+1. Download the global raster as single file at [https://cidportal.jrc.ec.europa.eu/ftp/jrc-opendata/GHSL/GHS_POP_MT_GLOBE_R2019A/GHS_POP_E2015_GLOBE_R2019A_54009_1K/V1-0/GHS_POP_E2015_GLOBE_R2019A_54009_1K_V1_0.zip](https://cidportal.jrc.ec.europa.eu/ftp/jrc-opendata/GHSL/GHS_POP_MT_GLOBE_R2019A/GHS_POP_E2015_GLOBE_R2019A_54009_1K/V1-0/GHS_POP_E2015_GLOBE_R2019A_54009_1K_V1_0.zip)
+2. Extract archive
+3. Move `GHS_POP_E2015_GLOBE_R2019A_54009_1K_V1_0.tif` to the OQT data directory
+
+
+### GHS-SMOD R2019A
+
+Information:
+- https://ghsl.jrc.ec.europa.eu/ghs_smod2019.php
+- product: GHS-SMOD, epoch: 2015, resolution: 1 km, coordinate system: Mollweide.
+
+Setup steps:
+1. Download the global raster as single file at [https://cidportal.jrc.ec.europa.eu/ftp/jrc-opendata/GHSL/GHS_SMOD_POP_GLOBE_R2019A/GHS_SMOD_POP2015_GLOBE_R2019A_54009_1K/V2-0/GHS_SMOD_POP2015_GLOBE_R2019A_54009_1K_V2_0.zip](https://cidportal.jrc.ec.europa.eu/ftp/jrc-opendata/GHSL/GHS_SMOD_POP_GLOBE_R2019A/GHS_SMOD_POP2015_GLOBE_R2019A_54009_1K/V2-0/GHS_SMOD_POP2015_GLOBE_R2019A_54009_1K_V2_0.zip)
+2. Extract archive
+3. Move `GHS_SMOD_POP2015_GLOBE_R2019A_54009_1K_V2_0.tif` to the OQT data directory
+
+
+### Nighttime Lights VIIRS - Annual VNL V2
+
+Information:
+- https://eogdata.mines.edu/products/vnl/
+- product: Annual VNL V2, resolution: 15 arc second (~500m at the Equator), coordinate system: EPSG:4326 (Geographic Latitude/Longitude)
```

Comment:
maybe you could add here that we are using the 2020 version, because there are also files for the past years and probably there will be files for future years as well.

Reply:
Done